src/test/java/core/basesyntax/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.exception.WritingException;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FileWriterServiceImplTest {
+    private FileWriterServiceImpl fileWriterService = new FileWriterServiceImpl();
+
+    @Test
+    void write_nullData_NotOk() {
+        String fileName = "name";
+        assertThrows(WritingException.class, () -> fileWriterService.write(fileName, null));
+    }
+
+    @Test
+    void write_okData_Ok() {
+        String fileName = "/Users/macbook/IdeaProjects/jv-fruit-shop-tests/src/test/resources/name";
```

Comment:
it is the problem of your failed CI. Make relative path, /Users/macbook/IdeaProjects/ does not exist on git